java/animate-3d-graphic/src/main/java/com/esri/arcgisruntime/animate3dgraphic/MainActivity.java

```diff
@@ -0,0 +1,462 @@
+package com.esri.arcgisruntime.animate3dgraphic;
```

Comment:
Also should the folder structure include folder 'sample' in it? similar to other samples example,
java.com.esri.arcgisruntime.sample.animate3dgraphic.

Reply:
Good catch! Thanks!